src/icons/fillInfo.svg

```diff
@@ -0,0 +1,77 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M155.811 167.163C155.981 166.925 156.239 166.765 156.528 166.717C156.817 166.67 157.113 166.739 157.35 166.909C157.588 167.08 157.749 167.338 157.796 167.627C157.844 167.916 157.775 168.212 157.604 168.45L150.46 178.414L150.457 178.417C150.363 178.544 150.242 178.649 150.104 178.725C149.965 178.802 149.812 178.848 149.655 178.861C149.497 178.874 149.338 178.853 149.189 178.8C149.041 178.747 148.905 178.663 148.791 178.553L148.788 178.55L145.972 175.733C145.869 175.63 145.788 175.509 145.733 175.375C145.677 175.241 145.649 175.097 145.649 174.952C145.649 174.807 145.677 174.664 145.733 174.53C145.788 174.396 145.869 174.274 145.972 174.172C146.074 174.069 146.196 173.988 146.33 173.933C146.464 173.877 146.607 173.849 146.752 173.849C146.897 173.849 147.041 173.877 147.175 173.933C147.309 173.988 147.43 174.07 147.533 174.172L149.427 176.069L155.811 167.163Z" fill="white"/>
+<rect x="166.811" y="96.892" width="29.1892" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<rect x="166.811" y="104.676" width="50.5946" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<rect x="166.811" y="131.919" width="29.1892" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<rect x="166.811" y="139.703" width="50.5946" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<rect x="166.811" y="166.946" width="29.1892" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<rect x="166.811" y="174.73" width="50.5946" height="3.89189" rx="1.94595" fill="#EBECF0"/>
+<path d="M199.75 50.4C196.989 50.4 194.83 48.0974 193.899 45.4978C193.194 43.5315 192.068 41.7247 190.576 40.2177C187.904 37.5171 184.279 36 180.5 36C176.721 36 173.096 37.5171 170.424 40.2177C168.932 41.7247 167.806 43.5315 167.101 45.4978C166.17 48.0974 164.011 50.4 161.25 50.4H148.974C145.551 50.4 143.14 53.7629 144.239 57.0052L148.849 70.6052C149.537 72.6346 151.442 74 153.585 74H207.415C209.558 74 211.463 72.6346 212.151 70.6052L216.761 57.0052C217.86 53.7629 215.449 50.4 212.026 50.4H199.75Z" fill="url(#paint0_linear)"/>
+<circle cx="180.5" cy="47.5" r="4.78788" fill="white" stroke="#19194B" stroke-width="3.42424"/>
+<rect x="160" y="59" width="41" height="2" rx="1" fill="#F4F5F7"/>
+<rect x="165" y="63" width="32" height="2" rx="1" fill="#F4F5F7"/>
+<rect x="169" y="67" width="24" height="2" rx="1" fill="#F4F5F7"/>
+<g filter="url(#filter0_d)">
+<mask id="path-28-outside-1" maskUnits="userSpaceOnUse" x="147.803" y="132.719" width="135.499" height="124.005" fill="black">
```

Comment:
This id must be updated to be unique, such as: `id="fill-info-icon-path-28-outside-1"` and then all references `#path-28-outside-1` must be updated as well to: `#fill-info-icon-path-28-outside-1`. Same for all other `id="` attributes in this file. Thanks!

Reply:
Just as a curiosity, why do the id's have to be unique?

Reply:
Good question! it's because the HTML DOM always expects that there will be only *one* occurrence per ID, and when we have multiple SVG icons with the same ID(s) then the browser simply picks the very first one it saw and uses that, which causes rendering problems for our icons (for example: if we have an orange icon first in the DOM, then all subsequent icons that have the same IDs will render in orange even when they shouldn't). This was a hard problem for us to discover...